docs/streams/developer-guide.html

```diff
@@ -2245,7 +2249,10 @@
             .groupBy((key, word) -> word, Serialized.with(stringSerde, stringSerde));
 
           // Create a window state store named "CountsWindowStore" that contains the word counts for every minute
-          groupedByWord.count(TimeWindows.of(60000), "CountsWindowStore");
+          groupedByWord.windowedBy(TimeWindows.of(60000))
+            .count(Materialized.&lt;String, Long, WindowStore&lt;Bytes, byte[]&gt;&gt;as("CountsWindowStore")
+                withKeySerde(Serdes.String()); // value serde will automatically set to Serdes.Long()
```

Comment:
nit: `count() sets value serde to Serdes.Long() automatically`

Reply:
And if you don't like to update with @mjsax's suggestion, you'll need to fix the grammar in the current comment. ;-)